<launch>
  <arg name="image_topic" />
  <arg name="output_image_topic" />
  <!-- Start openpose_ros -->
  <node pkg="openpose_ros" type="openpose_ros_node" name="openpose_ros_node">
    <param name="image_topic"   	value="$(arg image_topic)" />
    <param name="output_topic"      value="$(arg output_image_topic)" />
    <param name="output_topic_video"      value="/openpose_ros/human_list" />
  </node>
</launch>
